keras_cv/layers/__init__.py

```diff
@@ -17,6 +17,10 @@
 from tensorflow.keras.layers import RandomWidth
 
 from keras_cv.layers.augmenter import Augmenter
+from keras_cv.layers.detectron2_layers import AddPositionalEmbedding
```

Comment:
Do we want to put these under a `detectron2` namespace?

Reply:
Done.